static/chat.js

```diff
@@ -112,7 +112,7 @@ socket.on("private_data", (data) => {
     // update the url when the room is changed.
     // window.history.replaceState({"pageTitle": `${data['name']} - Chat`}, "", `/chat/${data['name']}`);
     // roomname = document.getElementById("RoomDisplay").innerHTML = '/'+data['name'];
-    window.history.replaceState({"pageTitle": `Private Chat`}, "", `/chat/${data['name']}`);
+    window.history.replaceState({"pageTitle": `Private Chat`}, "", `/Private`);
```

Comment:
well, if your going to do this, might aswell include the other person's displayname in the uri (like what we do with rooms)